Application/EdFi.Ods.SandboxAdmin.Web/App_Start/AppStartUp.cs

```diff
@@ -45,39 +52,37 @@ protected void Application_Start()
             RouteConfig.RegisterRoutes(RouteTable.Routes);
             BundleConfig.RegisterBundles(BundleTable.Bundles);
 
-            AuthConfig.RegisterAuth();
-
             BootstrapContainer();
 
             GlobalFilters.Filters.Add(new SetCurrentUserInfoAttribute(() => _container.Resolve<ISecurityService>()));
 
             _databaseEngine = _container.Resolve<IApiConfigurationProvider>().DatabaseEngine;
 
-            if (_databaseEngine == DatabaseEngine.SqlServer)
-            {
-                InitializeWebSecurity();
-            }
-
             BootstrapHangfire();
-        }
 
-        private BackgroundJobServer _backgroundJobServer;
+            appBuilder.UseCookieAuthentication(new CookieAuthenticationOptions
+            {
+                AuthenticationType = DefaultAuthenticationTypes.ApplicationCookie,
+                LoginPath = new PathString("/Account/Login")
+            });
 
-        private DatabaseEngine _databaseEngine;
 
-        protected void Application_PreSendRequestHeaders()
```

Comment:
do we need to apply a middleware to string these headers?

Reply:
created ODS-4431 to address after net core conversion